griptape/config/azure_openai_structure_config.py

```diff
@@ -40,7 +40,9 @@ class AzureOpenAiStructureConfig(StructureConfig):
     api_key: Optional[str] = field(kw_only=True, default=None, metadata={"serializable": False})
     prompt_driver: BasePromptDriver = field(
         default=Factory(
```

Comment:
Do we need to use attr's `Factory` now that we're doing our own? Can we somehow extend/modify the existing `Factory`?